src/@sdk/api/index.ts

```diff
@@ -59,18 +66,18 @@ export class SaleorAPI {
       jobsManager
     );
     this.cart = new SaleorCartAPI(
-      checkoutRepositoryManager,
-      networkManager,
+      localStorageManager,
+      apolloClientManager,
       saleorState,
       loadOnStart.cart,
       jobsManager
     );
 
     this.legacyAPIProxy.attachAuthListener(authenticated => {
       if (!authenticated) {
-        repository.setCheckout({});
-        repository.setPayment({});
-        repository.setJobs(null);
+        localStorageHandler.setCheckout({});
+        localStorageHandler.setPayment({});
+        localStorageHandler.setJobs(null);
```

Comment:
Can we get unified `.clear()` method? ATM I would not know if I should pass null or empty dict to the `set`.

Reply:
Yes, we can, but I'm going to remove completely this `attachAuthListener` from here in another pull request #727. May we fix it in that next PR?

Reply:
Exactly [here](https://github.com/mirumee/saleor-storefront/pull/727/files#diff-4fe31a9e3e6902aa82e1cf598a955343R98)

Reply:
If thats going to be removed anyway then okay